src/nocmodl/nocpout.cpp

```diff
@@ -256,8 +256,6 @@ void parout() {
 \n#if !NRNGPU\
 \n#undef exp\
 \n#define exp hoc_Exp\
-\n#undef pow\
-\n#define pow hoc_pow\
```

Comment:
please revert and create a separate pr, because it's not trivial to decide.

Reply:
did